bindings/ruby/lib/typelib/clang.rb

```diff
@@ -29,6 +29,13 @@ def self.load(registry, file, kind, options)
             
             include_dirs = options[:include_paths] || []
             include_path = include_dirs.map { |d| "-I#{d}" }
+            defines = options[:define] || []
+            #add -D to the defines, if its missing
+            defines.each do |define|
+                if not define.start_with? '-D'
+                    define.prepend('-D')
```

Comment:
avoid changing values in-place unless it's required by e.g. performance reasons, e.g. the whole loop would be best written

```
defines = defines.map { |d| "-D#{d}" }
```